### PR TITLE
Feature/proj to proj transitions

### PR DESCRIPTION
**Pull request by joshmurr**

### CODE REVIEW

File: src/components/Carousel.tsx [hunk 1]
```diff
@@ -67,18 +63,15 @@ export function Carousel({ items }: { items: string[] }) {
             key={i}
             href={`/work/item-${i}`}
             onClick={() => dispatch({ type: "projectIndex", value: i })}
-            initial={{ opacity: state.projectIndex === i ? 1 : 0 }}
-            exit={{ opacity: state.projectIndex === i ? 1 : 0 }}
-            transition={transition}
-            animate={{ opacity: 1 }}
           >
-            <motion.img
+            <MotionImage
```
Comment (piersolenski) [inline src/components/Carousel.tsx:67]: Can we convert to a wrapper for any media?

File: src/components/Image.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+const Wrapper = styled(motion.div)({});
+
+export const MotionImage = forwardRef<HTMLDivElement, Props>(
+  ({ src, priority = false, ...props }, ref) => {
+    return (
+      <Wrapper ref={ref} transition={transition} {...props}>
+        <NextImage
+          src={src}
+          alt="Bum"
```
Comment (piersolenski) [inline src/components/Image.tsx]: Alt tag appreciated 🙏

File: src/styles/globals.css [hunk 3]
```diff
@@ -7,7 +7,7 @@
   --background-start-rgb: 214, 219, 220;
   --background-end-rgb: 255, 255, 255;
 
-  --card-size: 50vh;
+  --card-size: 500px;
```
Comment (piersolenski) [inline src/styles/globals.css]: Did you change it to pixels for a reason? I remember before setting it to pixels fixed some stuff for me, but can we continue to test with 50vh, as that is how the current cards work so will need to be the same eventuallly...

**Review by piersolenski — CHANGES_REQUESTED** [review]

**Comment by piersolenski** [comment]
@joshmurr this looks like it's almost there 🤞 💪 the animation back to the homepage for latter cards is broken atm though 🙈

File: src/styles/globals.css (hunk 3 above)
Comment (joshmurr) [reply src/styles/globals.css]: I set it to 500px cos we need to provide a width and height (in px) to Next/Image, so thought I'd match them. Just defaulting to 500px in the demo so we don't have to bother with `useNextSanityImage` or something to get attributes for the images.

That said it still works with `50vh` so I'll revert. But something to remember.

**Comment by joshmurr** [comment]
Anim back to homepage should be good. Just wasn't updating the project ID in global state when moving from proj-to-proj so the carousel scrolling was broken.

**Pull request merged**